src/main/java/pl/piekoszek/gorskimatches/facebook/FacebookRequestHandler.java

```diff
@@ -0,0 +1,29 @@
+package pl.piekoszek.gorskimatches.facebook;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class FacebookRequestHandler {
+
+    private final FacebookCommands commands;
+
+
+    public FacebookRequestHandler(FacebookCommands commands) {
+        this.commands = commands;
+    }
+
+    void handle(FacebookHookRequest request) {
+        FacebookEntry facebookEntry = new FacebookEntry();
+        request.getEntry().forEach(entry -> entry.getMessaging().forEach(message
+                -> {
+            facebookEntry.setId(message.getSender().get("id"));
+            commands.helloMessage(message.getMessage(), facebookEntry.getId());
+            commands.quiz(message.getMessage(), facebookEntry.getId());
```

Comment:
`FacebookRequestHandler` has to know about every possible command in `FacebookCommands`, I'd rather have it inside `FacebookCommands`